microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

```diff
@@ -74,6 +74,27 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
 ![A possible issue being explained on an output element](microsoft-edge-devtools-extension-images/live-issue-reporting.png)
 
 
+#### Automated Quick Fixes and issue filtering
+
+Using Quick Fixes, you can customize the error reporting of the extension to the needs of the current project.
+
+When you hover over an element that has an issue, you get a lightbulb icon indicating that there are Quick Fixes available:
+
+![An anchor element with a protocol-specific href attribute highlighted as a problem, by a wavy underline and lightbulb icon above it](microsoft-edge-devtools-extension-images/light-bulb.png)
+
+Clicking the lightbulb icon shows a list of options. For example, if you added a link with a protocol-relative URL, you get the following list of Quick Fixes to choose from:
+
```

Comment:
rename the png from quick-view-options.png to quick-fix-options.png

Reply:
"rename the png from quick-view-options.png to quick-fix-options.png" - done locally in new forked branch user/mikehoffms/exten-img/cln